tests/test_redirects.py

```diff
@@ -86,37 +58,18 @@ def test_that_checks_redirect_using_locales_and_os(
 
         parsed_url = urlparse(response.url)
 
-        Assert.equal(
-            response.status_code,
-            requests.codes.ok,
-            'Redirect failed with HTTP status %s. \n %s' %
-            (response.status_code, self.response_info_failure_message(
-                base_url,
-                param,
-                response))
-        )
-        Assert.equal(
-            parsed_url.scheme,
-            'http',
-            'Failed by redirected to incorrect scheme %s. \n %s' %
-            (parsed_url.scheme, self.response_info_failure_message(
-                base_url,
-                param,
-                response))
-        )
-        Assert.not_equal(
-            self.get_x_backend_server(response),
-            'Unknown',
-            'Failed, x-backend-server was not in the response object %s' %
-            (self.response_info_failure_message(base_url,
-                param,
-                response))
-        )
-
-    # xfail as mentioned in https://github.com/mozilla/bouncer-tests/issues/46
-    @pytest.mark.xfail(
-        "config.getvalue('base_url') == 'http://download.allizom.org'"
-    )
+        assert (requests.codes.ok == response.status_code,
+                'Redirect failed with HTTP status %s. \n %s' %
+                (response.status_code, self.response_info_failure_message(base_url, param, response)))
+
+        assert ('http' == parsed_url.scheme,
+               'Failed to redirect to the correct scheme. %s' %
+                (self.response_info_failure_message(base_url, param, response)))
```

Comment:
Nit: No need to put the string formatting argument in brackets when there's only one.